Create ui-c8y-1022-47-2-modbus-protocol-add-holding-register-min-and-max-values-should-allow.md (#3892)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1022-47-2-modbus-protocol-add-holding-register-min-and-max-values-should-allow.md

---
date: ""

> title: Improved Value Normalisation for Modbus Registers

Reviewer: ```suggestion
title: Improved value normalization for Modbus registers
```

> title: Modbus protocol add holding register min and max values should allow floating point numbers (#10165) [GRAFT][release/cd] (#10250)

Reviewer 2: ```suggestion
title: Improved Value Normalisation for Modbus Registers
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5012
version: 1022.47.2
---

> Co-authored-by: Dominik Schmitz <dominik.schmitz@cumulocity.com>

Reviewer 2: ```suggestion
In the Device Management app, it is now possible to enter decimal values as minimum and maximum for Modbus register value normalisation. Previously, the interface suggested that values like 655.1 were allowed, but only whole numbers could be saved, which prevented some device configurations. With this update, the app accepts floating point numbers in all relevant fields for Cloud fieldbus types, including Modbus. This ensures that register settings can match the capabilities of your devices and allows for more accurate value scaling.
```

> In the Device Management app, it is now possible to enter decimal values as minimum and maximum for Modbus register value normalisation. Previously, the interface suggested that values like 655.1 were allowed, but only whole numbers could be saved, which prevented some device configurations. With this update, the app accepts floating point numbers in all relevant fields for Cloud fieldbus types, including Modbus. This ensures that register settings can match the capabilities of your devices and allows for more accurate value scaling.

Reviewer: ```suggestion
In the Device Management application, it is now possible to enter decimal values as minimum and maximum values for Modbus register value normalization. Previously, values like 655.1 were allowed in the UI, but only whole numbers were saved, which prevented exact device configurations. With this change, the application accepts floating point numbers in all relevant fields for Cloud fieldbus types, including Modbus. This ensures that register settings can match the capabilities of your devices and allows for more accurate value scaling.
```
